build.gradle.kts

```diff
@@ -467,10 +467,10 @@ tasks.register("createDmg") {
 fun genBuildInfo(buildInfoFilePath: String) {
   val now = Date()
   val nowIso = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZ").format(now)
-  val branchName = runCommand(listOf("git", "-C", "$projectDir", "rev-parse", "--abbrev-ref", "HEAD"),
+  val branchName = runCommand(listOf("git", "-C", "${projectDir}", "rev-parse", "--abbrev-ref", "HEAD"),
```

Comment:
I insist we replace `"${projectDir}"` by dropping quotes and make explicit conversion to String, as it would make developer intent more clear for future code readers. This is more clear with Java's `"" + foo"` but `"${projectDir}"` looks like common newbie bug and someone in future might be tempted to "fix that" and then waste some time.

Reply:
Fair enough. I changed it to use toString().